api/rpc/stack/stack.go

```diff
@@ -119,7 +126,9 @@ func (s *Server) processService(ctx context.Context, stack *Stack, serv *service
 	request := &service.ServiceCreateRequest{
 		ServiceSpec: serv,
 	}
-	server := service.Service{}
+	server := service.Service{
```

Comment:
You probably don't need to do this (go through the rpc interface). You already call [service.Remove](https://github.com/appcelerator/amp/blob/master/api/rpc/service/service.go#L160) directly, so you could probably call [service.Create](https://github.com/appcelerator/amp/blob/master/api/rpc/service/service.go#L61) directly as well. (We can refactor service.Create to be a bit more straightforward -- like service.Remove now or sometime later...)